Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -159,6 +159,11 @@ int LayoutAnimationsManager::findPrecedingViewTagForTransition(const int tag) {
   return -1;
 }
 
+const std::vector<int> LayoutAnimationsManager::getSharedGroup(const int tag) {
```

Comment:
Since the `std::vector` is stored somewhere, can we return a reference instead of a copy and is it safe to do so?
```suggestion
const std::vector<int> &LayoutAnimationsManager::getSharedGroup(const int tag) {
```